lib/sms_text.rb

```diff
@@ -0,0 +1,17 @@
+require 'rubygems'
+class Sms_Text
+
+	def send_text
+		account_sid = 'AC1cffe35eb134133af61d8ab716f40588'
+		auth_token = '48e47a677c14dd9162f479523ed7fb3c'
+
+		@client = Twilio::REST::Client.new account_sid, auth_token
```

Comment:
This should be injected in to make the SMS class reusable - what if you change SMS provider? Currently you would have to rewrite this entire class.